tests/pactHelper.js

```diff
@@ -501,6 +501,11 @@ const getProviderBaseUrl = function () {
   return providerBaseUrl
 }
 
+const getMockServerBaseUrl = function () {
```

Comment:
you need to call this function in here at very first 
https://github.com/owncloud/owncloud-sdk/blob/4a996e8f2e0b25dd461cf8498c31ceb6c145ec5c/tests/pactHelper.js#L131-L142

```js
...
 baseUrl: getMockServerBaseUrl(),
...
```